### PR TITLE
Fix/job template refinements

### PR DESCRIPTION
**Pull request by ximekilgsa**
## Changes proposed in this pull request

- Fixes some instructions for content editors
- Creates template parts for open and upcoming listings
- Adds new logic that displays info sessions based on date AND time, not just date
- Adds JS to hide info sessions without needing a deployment, since they happen at different times.
- Removes an undisplayed image with an incorrect path that was throwing an error in the console

## security considerations

N/A

### CODE REVIEW

**Review by wesley-dean-gsa — COMMENTED** [review]
Acknowledged that we're supporting 12-hour, am/pm time so this isn't a drop-in replacement:

https://github.com/moment/luxon/blob/master/docs/parsing.md

File: _includes/scripts.html [hunk 1]
```diff
@@ -45,3 +45,18 @@
       });
   });
 </script>
+
+<!-- Conditionally hide past info sessions -->
+<script>
+  function hidePastInfoSessions() {
+    const now = new Date().getTime();
```
Comment (wesley-dean-gsa) [inline _includes/scripts.html:52]: do we need to translate this into `America/New_York` ?

File: _includes/scripts.html [hunk 2]
```diff
@@ -45,3 +45,18 @@
       });
   });
 </script>
+
+<!-- Conditionally hide past info sessions -->
+<script>
+  function hidePastInfoSessions() {
```
Comment (wesley-dean-gsa) [inline _includes/scripts.html:51]: I really, really like this approach.

File: .eleventy.js [hunk 3]
```diff
@@ -155,8 +155,38 @@ module.exports = function (config) {
     });
   }
 
-  config.addLiquidShortcode("getStateFromDates", getStateFromDates);
+  // Get Date and Time as Seconds
+  // Datetime format: YYYY-MM-DD HH:
+  config.addLiquidShortcode("getDateTimeinSeconds", getDateTimeinSeconds);
+  function getDateTimeinSeconds(datetime) {
```
Comment (wesley-dean-gsa) [inline .eleventy.js:161]: I thought I put a comment in about this but I don't see it any more, so if it's a duplicate, please forgive me.

Acknowledged that we're supporting 12-hour times, so this likely isn't a drop-in replacement:

https://github.com/moment/luxon/blob/master/docs/parsing.md#iso-8601
https://github.com/moment/luxon/blob/master/docs/formatting.md#unix-timestamps

File: _includes/layouts/jointts/home.html [hunk 4]
```diff
@@ -27,133 +27,20 @@ <h1>{{ title }}</h1>
     </div>
     <div class="grid-row grid-gap">
       <div class="desktop:grid-col-8 usa-prose">
+        <!-- Open Jobs -->
         <h2 id="open-positions">Open positions</h2>
         <p>We are hiring and will be sharing upcoming jobs and open positions as they are available.</p>
+        {% include "layouts/jointts/open-jobs.html" %}
```
Comment (wesley-dean-gsa) [inline _includes/layouts/jointts/home.html:33]: love this, not just here, but all of the presentation template "chunks"

File: pages/jointts/positions/Presidential-Innovation-Fellows.md [hunk 5]
```diff
@@ -117,14 +117,14 @@ key_objectives:
 # are TWO digits 
 info_sessions:
   - link: https://gsa.zoomgov.com/webinar/register/WN_pEhDPgcHRGKxOOlQr4S-Cw#/registration
-    date: 2024-09-26
+    date: 2024-09-25
     headline: Wednesday, September 25th, 2024
     time: 1:00-2:00pm ET (10:00am-11:00am PT)
```
Comment (wesley-dean-gsa) [inline pages/jointts/positions/Presidential-Innovation-Fellows.md:122]: would it be possible to set this up to be similar to the `opens` and `closes` we're using for job postings?  if so, would it make sense to do the calculation automatically?

Comment (ximekilgsa) [reply pages/jointts/positions/Presidential-Innovation-Fellows.md:122]: opens and closes is formatted the same as this...!

File: .eleventy.js (hunk 3 above)
Comment (ximekilgsa) [reply .eleventy.js:161]: Is there something actionable here or can I resolve the comment?

File: pages/jointts/positions/Presidential-Innovation-Fellows.md (hunk 5 above)
Comment (ximekilgsa) [reply pages/jointts/positions/Presidential-Innovation-Fellows.md:122]: @wesley-dean-gsa Did you mean in terms of not needing the headline field?

**Review by wesley-dean-gsa — APPROVED** [review]

**Pull request merged**